samples/automl/automlNaturalLanguageModel.js

```diff
@@ -0,0 +1,599 @@
+  };
+
+  // Create a model with the model metadata in the region.
+  client
+    .createModel({parent: projectLocation, model: myModel})
+    .then(responses => {
+      const operation = responses[0];
+      const initialApiResponse = responses[1];
+
+      console.log(`Training operation name: ${initialApiResponse.name}`);
+      console.log(`Training started...`);
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      const model = responses[0];
+
+      // Retrieve deployment state.
+      var deploymentState = ``;
+      if (model.deploymentState === 1) {
+        deploymentState = `deployed`;
+      } else if (model.deploymentState === 2) {
+        deploymentState = `undeployed`;
+      }
+
+      // Display the model information.
+      console.log(`Model name: ${model.name}`);
+      console.log(`Model id: ${model.name.split(`/`).pop(-1)}`);
+      console.log(`Model display name: ${model.displayName}`);
+      console.log(`Model create time:`);
+      console.log(`\tseconds: ${model.createTime.seconds}`);
+      console.log(`\tnanos: ${model.createTime.nanos}`);
+      console.log(`Model deployment state: ${deploymentState}`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_natural_language_createModel]
+}
+
+function getOperationStatus(operationFullId) {
+  // [START automl_natural_language_getOperationStatus]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const operationFullId = `Full name of an operation, eg. “Projects/<projectId>/locations/us-central1/operations/<operationId>
+  // Get the latest state of a long-running operation.
+
+  // Get the latest state of a long-running operation.
+  client.operationsClient.getOperation(operationFullId).then(responses => {
+    const response = responses[0];
+    console.log(`Operation status: ${response}`);
+  });
+  // [END automl_natural_language_getOperationStatus]
+}
+
+function listModels(projectId, computeRegion, filter_) {
+  // [START automl_natural_language_listModels]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const filter_ = `filter expressions, must specify field, e.g. “imageClassificationModelMetadata:*”`;
+
+  // A resource that represents Google Cloud Platform location.
+  const projectLocation = client.locationPath(projectId, computeRegion);
+
+  // List all the models available in the region by applying filter.
+  if (filter_ === ``) filter_ = `textClassificationModelMetadata:*`;

```

Comment:
Can we set `filter_`'s default value to something falsy (e.g. `null` or `undefined`) and do `filter_ = filter_ || 'textClassificationModelMetadata:*';` instead?